lib/src/io/node.dart

```diff
@@ -187,32 +223,44 @@ T _systemErrorToFileSystemException<T>(T callback()) {
   }
 }
 
-final stderr = Stderr(process.stderr);
+Stderr get stderr {
+  var stderr_ = process?.stderr;
+  if (stderr_ == null) {
+    throw UnsupportedError("stderr is only supported on Node.js");
+  }
+  return Stderr(stderr_);
+}
```

Comment:
This will create a new `Stderr` object every time it's referenced, rather than initializing it once and re-using it. You can put complex logic in a one-time initializer using an IIFE:

```suggestion
final Stderr stderr = () {
  var stderr_ = process?.stderr;
  if (stderr_ == null) {
    throw UnsupportedError("stderr is only supported on Node.js");
  }
  return Stderr(stderr_);
}();
```

Reply:
Fixed in Fixed in https://github.com/oddbird/dart-sass/commit/1937affd6fa9fc3965c72222f323f56fb53352d7